AY2021S1-CS2103-T16-3/tp
# Modify history behaviour, DG and UG fixes

### What this does
This PR changes the behaviour of the `history` command. Previously, the command listed previously entered commands in reverse chronological order. However, it now lists previously entered commands in chronological order, along with a positive integer before each command which indicates its position in the execution sequence (e.g. `3    clear` means that the command `clear` was the 3<sup>rd</sup> to be executed). Meanwhile, the displaying of commands in the `CommandBox` remain unaltered (only the commands themselves are listed). The reason for this change is to allow for a more detailed listing of previously entered commands which may provide to be more intuitive, resembling the output provided by the bash history command. 

The PR also updates the UG, DG as well as fix some failing tests based on the above change in behaviour. Additionally, the dependency on `KeyboardShortcutsEnum` for `CommandBox` is reflected as well. 



### How to test

1. Launch the application.
2. Type in a few commands
3. Ensure that the result output by `history` is in the format `n    command` where `n` is an integer.

### Notes

N/A
